src/store/actions.ts

```diff
@@ -36,14 +37,17 @@ export default function createActions(
 ): RootActions {
 	return {
 		async [ CREATE_LEXEME ]( { state, commit }: RootContext ): Promise<string> {
+			if ( !state.language || !state.lexicalCategory ) {
+				throw new Error( 'No language or lexical category!' ); // TODO
```

Comment:
What will happen when this is triggered? Because I'm not sure that T305854 will get done soon, given that it is in Milestone X.

Reply:
Nothing user-visible happens, because we don’t add a message to the store but the promise rejection is still caught.

I’m not sure what should happen. Previously, we would’ve tried to create a lexeme with the empty string as language or lexical category ID, which works fine in the dev entry point and produces an API error on the special page. Should we keep this behavior for now?

Reply:
Code-wise, I like this solution more. Though right now, people get at least a meaningful error message. OTOH, the current version isn't good enough anyway, so I think we can just go forward with this. Maybe we can increase the priority of that inline error-messages task soonish

Reply:
(GitHub showed me the PR approval but not this new comment… :rage:)

Okay, let’s leave it like this for the moment.